server/schedule/operator/step.go

```diff
@@ -643,9 +643,10 @@ func (dv DemoteVoter) String() string {
 }
 
 // ConfVerChanged returns the delta value for version increased by this step.
-func (dv DemoteVoter) ConfVerChanged(region *core.RegionInfo) bool {
-	peer := region.GetStoreLearner(dv.ToStore)
-	return peer.GetId() == dv.PeerID
+func (dv DemoteVoter) ConfVerChanged(region *core.RegionInfo) uint64 {
```

Comment:
`ConfVerChanged` will be called when the next steps are performed, and it is necessary to consider that the Peer may have been removed in the future.